wls-auth-service/src/main/resources/db/migrations/h2/V4_5__insertErgebnismeldungPermissions.sql

```diff
@@ -0,0 +1,103 @@
+-- awerte
```

Comment:
```suggestion
-- AWerte
```
damit es einheitlich ist, in den anderen files sind die kommentare auch groß geschrieben

Reply:
fixed mit 9b4a7f2edb9e41abb6192f3e5b1182c7f0cb61d8